docs/docs/reference/crd-reference/taskdefinition.md

```diff
@@ -32,27 +32,30 @@ differentiated by the `spec` section:
   [Synopsis for container-runtime](#synopsis-for-container-runtime)
   and
   [Examples for a container-runtime runner](#examples-for-a-container-runtime-runner).
-- Pre-defined containers
-
-    - Use the pre-defined `deno-runtime` runner
-      to define tasks using
-      [Deno](https://deno.com/)
-      scripts,
-      which use a syntax similar to JavaScript and Typescript,
-      with a few limitations.
-      You can use this to specify simple actions
-      without having to define a full container.
-      See
-      [Synopsis for Deno-runtime container](#deno-runtime)
-      and
-      [Deno-runtime examples](#examples-for-deno-runtime-runner).
-    - Use the pre-defined `python-runtime` runner
-      to define your task using
-      [Python 3](https://www.python.org/).
-      See
-      [Synopsis for python-runtime runner](#python-runtime)
-      and
-      [Examples for a python-runtime runner](#examples-for-a-python-runtime-runner).
+
+<!-- markdownlint-disable MD046 -->
+<!-- markdownlint-disable MD051 -->
```

Comment:
Let's try to remove the MD051 ignorer here as well

Reply:
Apologies for the delayed response.
I removed MD051 altogether and used html tags as suggested. Let's see if this passes now.